pyproject.toml

```diff
@@ -9,6 +9,7 @@ python = "^3.10"
 Django = "^4.1.3"
 djangorestframework = "^3.14.0"
 mysqlclient = "^2.1.1"
+django-webpack-loader = "^1.7.0"
```

Comment:
Again, about the dependency constraints. This time python. I think should be left unconstrained and dependabot should take care of the constraints.

Reply:
I think for now, we can leave those constraints and let depandabot deal with the later.